frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.scss

```diff
@@ -3,5 +3,7 @@
     float: right;
     align-self: flex-end;
     margin-top: -65px;
+    .monitoring-dashboards__dropdown-wrap {
+      margin-bottom: 5px;
+    }
```

Comment:
Personally, I am not a huge fan of using margins... this would be one reason why. Margins don't like to stay to their own element - they venture out and screw with the layout / rendering of parent elements.

I really think we should implement a `position: absolute` here as it removes it from the flow of the page -- which is what we are trying to do here.

Reply:
Whatever we do here is a bit of a hack as noted in the original PR that added these dropdowns.
The `float` is present because of how these dropdowns are used in the orignal monitoring dashboard page. They need to float to stay within the flow for responsive layout i believe.

For our purpose I agree that absolute positioning is fine.
Either way, this needs to be revisited because of the responsive layout issues we have with these dropdowns overlapping the tabs.

Reply:
this is simply overriding the current margin such that it doesn't overflow

Reply:
Fair enough, let us keep it this way then... the devil we know vs the devil we don't.